nextcord/embeds.py

```diff
@@ -517,40 +613,6 @@ def author(self) -> _EmbedAuthorProxy:
         """
         return EmbedProxy(getattr(self, "_author", {}))  # type: ignore
 
-    def set_author(
```

Comment:
Isn't this out of scope for this PR?

Reply:
Not sure what you mean? Outside of accidentally moving it I'm not sure what this comment is referring to

Reply:
Oh nevermind I didn't spot that it was moved